### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by shantanoo**
This branch is used for making miscellaneous changes like modifications suggested by pylint, removing unused imports, improving docstrings, etc. Am still working on this PR. Please do not merge.

Cc: @jcasner, @unixorn

### CODE REVIEW

**Comment by jcasner** [comment]
@shantanoo nice work so far on this PR - thanks for taking the initiative!

**Comment by unixorn** [comment]
Looks good so far.

File: infrastructure/infrastructure/utilities/path.py [hunk 1]
```diff
@@ -48,12 +47,12 @@ def rmrf(path, logger=None):
   you have to use different calls if a path points to a directory or file,
   which bloats the code.
 
-  @note: This behaves like `rm -rf` and will not complain if the directory
-  isn't empty, it will delete it and the contents anyway.
+  :note: This behaves like `rm -rf` and will not complain if the directory
+         isn't empty, it will delete it and the contents anyway.
 
-  @param path: path to file/directory you want deleted.
+  :param path: path to file/directory you want deleted.
```
Comment (jcasner) [inline infrastructure/infrastructure/utilities/path.py]: I have recently learned that you can (and should) include a type in the param string.  So, the format is:
`:param <type> <name>: <description> <example>`.  In this case, you could just do:
`:param str path: /path/to/delete`  
As long as we're updating, we should go ahead and make this change in the affected files.

**Comment by jcasner** [comment]
Given this is weeks old and not getting any work, i'm closing it.

**Pull request closed without merging**